joinHack.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require("mongoose");
+
+let conn = null;
+
+const uri = `mongodb://${process.env.MONGO_USER}:${process.env.MONGO_PASSWORD}@ds157136.mlab.com:57136/hackone`;
+
+export const join = async (event, context) => {
+  context.callbackWaitsForEmptyEventLoop = false;
+  const data = JSON.parse(event.body);
+  const { hackId, userId } = data;
+
+  const mongUserID = mongoose.Types.ObjectId(userId);
+
+  if (conn == null) {
+    conn = await mongoose.createConnection(uri, {
+      bufferCommands: false,
+      bufferMaxEntries: 0
+    });
+    conn.model(
+      "Hack",
+      new mongoose.Schema({team: Array})
+    );
+  }
+  const Hack = conn.model("Hack");
+
+  try {
+    let res = await Hack.findOneAndUpdate({_id: hackId}, {$push: {team: mongUserID } }, {new: true, upsert: true});
```

Comment:
this can be `const` right?

Reply:
Yeah it can, the model doesn't change however its property values will